### PR TITLE
DBDeploy improved splitting on --//@UNDO

### PR DESCRIPTION
**Pull request by azhurb**
Allow splitting on --//@UNDO with multiple spaces and without spaces.
Ignore tabs and spaces between -- and //@UNDO and any characters after @UNDO in that line.

### CODE REVIEW

File: classes/phing/tasks/ext/dbdeploy/DbDeployTask.php [hunk 1]
```diff
@@ -260,28 +260,31 @@ protected function generateSql($undo = false)
...
 
+                $deploySql = $split[0];
+                $undoSql = isset($split[1]) ? $split[1] : '';
+
                 if ($undo) {
-                    $sql .= substr($contents, $split + 10) . "\n";
+                    $sql .= $undoSql;
+                    $sql .= PHP_EOL;
                     $sql .= 'DELETE FROM ' . DbDeployTask::$TABLE_NAME . '
 	                         WHERE change_number = ' . $fileChangeNumber . '
```
Comment (kenguest) [inline classes/phing/tasks/ext/dbdeploy/DbDeployTask.php]: There are tabs here, please expand/replace them to spaces according to Coding Standards. Thanks!

Comment (azhurb) [inline classes/phing/tasks/ext/dbdeploy/DbDeployTask.php]: Done!

**Comment by azhurb** [comment]
@kenguest Is everything ok? I've replaced tabs.

**Comment by kenguest** [comment]
It looks good to me, but it's really down to @mrook who has merge access :-)

**Pull request merged**